qtox/build_qt_windows.sh

```diff
@@ -15,42 +15,80 @@ parse_arch --dep "qt" --supported "win32 win64" "$@"
 
 "$SCRIPT_DIR/download/download_qt.sh"
 
-OPENSSL_LIBS=$(pkg-config --libs openssl)
-export OPENSSL_LIBS
-
 CROSS_COMPILE="$MINGW_ARCH-w64-mingw32-"
 "${CROSS_COMPILE}gcc" --version
 
-./configure -prefix "$DEP_PREFIX" \
+mkdir qtbase/_build && pushd qtbase/_build
+../configure -prefix "$DEP_PREFIX" \
+  -appstore-compliant \
   -release \
-  -shared \
```

Comment:
What is the behavior if we don't specify "-shared"?

Reply:
Added it back. It's the same, but I removed it while testing -static and forgot to put it back.